hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/ethereum/EthTxSigsTest.java

```diff
@@ -96,26 +89,27 @@ public byte[] nextBytes(final int n) {
 
     @Test
     void signsLegacyUnprotectedNull() {
-        var tx =
+        final var tx =
                 new EthTxData(
                         null,
                         EthTxData.EthTransactionType.LEGACY_ETHEREUM,
                         null,
                         1,
-                        TINYBARS_57_IN_WEIBARS,
-                        TINYBARS_2_IN_WEIBARS,
-                        TINYBARS_57_IN_WEIBARS,
+                        TestingConstants.TINYBARS_57_IN_WEIBARS,
```

Comment:
Can we get back the imports so each constant need not be preceded with `TestingConstants.` ?